lib/_http_outgoing.js

```diff
@@ -512,8 +514,12 @@ function processHeader(self, state, key, value, validate) {
 }
 
 function storeHeader(self, state, key, value, validate) {
+
   if (validate)
     validateHeaderValue(key, value);
+  if (key.toLowerCase() === 'content-length') {
```

Comment:
```suggestion
  if (key.length === 14 && key.toLowerCase() === 'content-length') {
```

Reply:
Ok, will add it with the other changes and commit